README.md

```diff
@@ -44,3 +44,82 @@ along with this program.  If not, see https://www.gnu.org/licenses/
 ___
 
 Project Website: https://metrocs.github.io/imagelab/
+
+## Build Instructions 
+
+BUILD INSTRUCTIONS
+
+◾️Introduction
+
+This documents intent is to support developers in understanding how to 
+build the Image Lab project. This document offers support on key useful commands and error solutions that can make building Image Lab a smoother process.
+
+◾️Before You Build
+- Install Gradle
+ sudo apt install gradle
+- Install Canberra
+ Sudo apt install libcanberra-gtk-module libcanberra-gtk3-module
+
+
+◾️What is Gradle
+
+Gradle is a automated build system that makes working with large projects easier.
+It streams line the build process by skipping re-execution and allows developers to preform tasks suing one line of code in the command line.
+
+
+
+◾️How does Gradle Work?
+
+▪️Project 
+- A project is a depiction of what needs to be built and how we get an artifact at the end of a build.
+
+settings.gradle - Where projects are registered.
+build.gradle - constructs commands that can be used to execute tasks.
+
+
+▪️Task
+-A task is used to execute a action during the build process. For example, ./gradlew compileJava will compile the main class in the build. 
```

Comment:
This is the first place that "`./gradlew`" is introduced so it isn't clear what the "task" is in this example.